libs/datamodel/core/tests/render_to_dmmf/files/general.json

```diff
@@ -600,7 +597,7 @@
           "relationName": "AToB",
           "relationFromFields": [],
           "relationToFields": [],
-          "relationOnDelete": "NONE",
```

Comment:
Hmm interesting, so we are not returning this anymore in case of `None`? Could it break something typescript-side?

Reply:
Asked from the ts team.

Reply:
ping @millsp @williamluke4 @Jolg42 will we break some of your code?

Reply:
We'll need to remove it from our typings and update some snapshots. All good on TS side. 

@steebchen will also need to remove it from Go Client.
and @madebysid from Studio

Also, I see some projects also have it defined and will need to remove it, like Amplication.

Reply:
I don't use this atm so no problem for me

Reply:
I don't use it either, all good.

Reply:
Actually, nothing to change in TS since the field is not removed but not always passed
https://github.com/prisma/prisma/blob/7ac84453f9e096fbe6652bc0a68e2a47e15ba344/src/packages/generator-helper/src/dmmf.ts#L84

Because it's an optional field typed to string.